crowbar_framework/app/controllers/api/crowbar_controller.rb

```diff
@@ -42,6 +42,16 @@ def upgrade
     else
       render json: Api::Crowbar.upgrade
     end
+  rescue Crowbar::UpgradeStatus::StartStepRunningError,
+         Crowbar::UpgradeStatus::StartStepOrderError => e
```

Comment:
Can't we check for StandardError here, or better, for some common predecessor of these exceptions?